docs/01-Using-Fleet/standard-query-library/standard-query-library.yml

```diff
@@ -626,7 +626,7 @@ spec:
 apiVersion: v1
 kind: policy
 spec:
-  name: MDM Enrolled (macOS)
+  name: MDM Enrolled (macOS) (MDM Required)
```

Comment:
This query technically works even if you're not on MDM, as it will tell you you are not, so I think the additional label is superfluous on this one.

Reply:
Agreed! @eashaw  As we discussed today, this one is on hold until I chat with Mike T, but you can go ahead and remove the label for the MDM Enrolled policy.